docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -5,8 +5,9 @@ module.exports = function (grunt) {
 
   function getIncludeFiles() {
     const patterns = [
-      '**/*.scss',
-      'node_modules/decanter/scss',
+      'src/**/*.scss',
+      'node_modules/bourbon/core',
```

Comment:
can we write a note about why we have to explicitly add this here? (since decanter must import it like this) and the node_modules line below is apparently not enough context for that import

Reply:
Great idea. I'll add a comment above this line.